Sources/Workspace/Workspace.swift

```diff
@@ -2320,7 +2320,7 @@ extension Workspace {
             // way to get it back out of the resolver which is very
             // annoying. Maybe we should make an SPI on the provider for
             // this?
-            let container = try await { containerProvider.getContainer(for: package, skipUpdate: true, completion: $0) } as! RepositoryPackageContainer
+            let container = try tsc_await { containerProvider.getContainer(for: package, skipUpdate: true, completion: $0) } as! RepositoryPackageContainer
             guard let tag = container.getTag(for: version) else {
                 throw StringError("Internal error: please file a bug at https://bugs.swift.org with this info -- unable to get tag for \(package) \(version); available versions \(container.reversedVersions)")
             }
```

Comment:
@abertelrud @neonichu beyond the scope of this PR, but the fact the code [in the above three files] is blocking on what looks to be I/O seems like its worth a deeper look. at a high-level, "asynchronity" should bubble all the way to the "top" and only the UI/CLI call sites should block or use a dispatch queue to synchronize the "effects" where appropriate. Usage in tests is fine to simplify.